demos/python_demos/llm_text_generation/servable_unary/streamer.py

```diff
@@ -0,0 +1,83 @@
+#*****************************************************************************
+# Copyright 2024 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#*****************************************************************************
+
```

Comment:
Is it exact copy of the streamer.py from streaming endpoint implementation or does it differ somehow?

Reply:
It does not differ it is the same for both endpoints. The same goes for config.py. Maybe we should have one directory for both demos, and switch handler path in graph.pbtxt depending on the demo?  If so, I think it is out of scope of this task (actually a bugfix)